streams/test-utils/src/main/java/org/apache/kafka/streams/processor/MockProcessorContext.java

```diff
@@ -165,7 +165,7 @@ public long timestamp() {
          *
          * @return A key/value pair. Not null.
          */
-        @SuppressWarnings({"WeakerAccess", "unused"})
+        @SuppressWarnings({"WeakerAccess", "unused", "rawtypes"})
         public KeyValue keyValue() {
```

Comment:
Cannot change this to return type to `KeyValue<?, ?>` as it's public API and might break existing code if changed.